src/org/zalando/stups/magnificent/api.clj

```diff
@@ -136,6 +136,19 @@
       ring/response
       fring/content-type-json)))
 
+(defn get-team-or-account
+  [team-or-account request]
+  (try+
+    (:body (get-team {:team team-or-account} request))
+    (catch [:http-code 404] []
+      (try+
+        (let [account (:body (get-account {:type "aws" :account team-or-account} request))]
+          (log/warn "Team is actually an account: %s" {:team-or-account team-or-account})
+          account)
```

Comment:
Yes, but in case of `get-auth` we only need the `members` and that field is the same. It's named `get-team-or-account`, so users of this function should be prepared for either, I suppose.